src/main/java/mops/application/services/DatePollVoteService.java

```diff
@@ -23,13 +23,13 @@ public DatePollVoteService(DatePollRepository datePollRepository) {
     }
 
     @SuppressWarnings({"PMD.LawOfDemeter"})// Nullcheck
-    public DatePollBallot showUserVotes(UserId id, DatePollLink link) {
+    public DatePollBallot showUserVotes(UserId id, PollLink link) {
         final DatePoll poll = datePollRepository.load(link).orElseThrow();
         return poll.getUserBallot(id).orElseThrow();
     }
 
     @SuppressWarnings({"PMD.LawOfDemeter"})// Nullcheck
-    public void vote(DatePollLink link, UserId user, Set<DatePollEntry> yes, Set<DatePollEntry> maybe) {
+    public void vote(PollLink link, UserId user, Set<DatePollEntry> yes, Set<DatePollEntry> maybe) {
```

Comment:
hier ist die Methode wo 1. datepoll (wegen den entry counter) und 2. die m to n Beziehung die wir in datepollballot haben seperat abgespeichert werden muss